cmd/concourse/generate_key.go

```diff
@@ -74,3 +78,27 @@ func (cmd *GenerateKeyCommand) Execute(args []string) error {
 
 	return nil
 }
+
+type ExtractInternalConfigCommand struct {
+	FilePath string `short:"f"  long:"filename"  required:"true"  description:"File path where the key shall be created. When generating ssh keys, the public key will be stored in a file with the same name but with '.pub' appended."`
+
+	Seccomp bool `long:"seccomp" required:"false"  description:"Extract the default builtin seccomp filter"`
+}
+
+func (cmd *ExtractInternalConfigCommand) Execute(args []string) error {
+	var dest = cmd.FilePath
+	if cmd.Seccomp {
+		seccompfilter := bespec.GetDefaultSeccompProfile()
```

Comment:
So the whole point of this command is to print the `seccomp` variable from https://github.com/concourse/concourse/blob/master/worker/runtime/spec/seccomp.go in json format?

Reply:
Yes, because that's what then could be modified and re-used as baseline for custom trim down or extension based on use case and re-supplied to the containerd runner